src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -268,7 +270,10 @@ const SelfExclusion = (() => {
             return;
         }
         showFormMessage('Your changes have been updated.', true);
-        $('#ukgc_requirement_notice').setVisibility($('#exclude_until').attr('data-value'));
+        if ($('#exclude_until').attr('data-value')) {
+            $('#ukgc_requirement_notice').setVisibility(1);
+            $('#gamstop_info_bottom').setVisibility(0);
```

Comment:
can we hide this before we show the paragraph? then it will be hidden from the start instead of showing for a short time before hiding. (e.g. change lines 274 and 275)